libtcmu.c

```diff
@@ -277,78 +268,43 @@ static int generic_handle_cmd(struct tcmu_device *dev,
 	case MODE_SELECT:
 	case MODE_SELECT_10:
 		return tcmu_emulate_mode_select(cdb, iovec, iov_cnt, sense);
+	default:
+		tcmu_err("unknown command %x\n", cdb[0]);
+		return TCMU_NOT_HANDLED;
+	}
+}
+
+static bool command_is_store_call(uint8_t cmd)
+{
```

Comment:
IMO, Mike's approach fits well with the async call mechanism, not that command passthrough  doesn't fit well here. The *only* extra operation with command passthrough is an extra asynchronous call to `->handle_cmd` when the command would not be handled.